content/reference/admin/private_locations/control_plane_example_ecs/index.md

```diff
@@ -0,0 +1,199 @@
+---
+title: "Deploying a Control Plane on ECS"
+description: "How to deploy a Gatling Control Plane on AWS using Amazon Elastic Container Service (ECS) and Amazon Fargate"
+lead: "Run a Control Plane on AWS using Amazon Elastic Container Service (ECS) and Amazon Fargate"
```

Comment:
Does adding the Amazon word really help?

Reply:
Mention Private Locations and the ability to run load generators in private network. This helps SEO.

Reply:
> Does adding the Amazon word really help?

I copied the full names of the services as they appear on the AWS website, but probably not needed. AWS is already mentioned in the sentence anyway.

> Mention Private Locations and the ability to run load generators in private network.

👍